.github/actions/build-deps/action.yml

```diff
@@ -21,6 +21,11 @@ inputs:
     description: "The logging verbosity."
     required: false
     default: "verbose"
+  sanitizers:
+    description: "The sanitizers to enable ('None', 'Address', 'Thread')."
```

Comment:
This creates an impression that it's only Address or Thread sanitizer, but in reality, it's Address+Undefined or Thread+Undefined